corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -391,6 +405,7 @@ def tearDownClass(cls):
         super(IndicatorPillowTest, cls).tearDownClass()
 
     def tearDown(self):
+        delete_all_repeat_records()
```

Comment:
why is this needed?
wont any new records created during test be removed at the end of the test?